macros/shared

```diff
@@ -225,6 +244,12 @@ CROSS_COMPILATION_CONF_EOF\
 %cross_scan_attribution \
   bottlerocket-license-scan --spdx-data /usr/libexec/tools/spdx-data --out-dir %{buildroot}%{_cross_attribution_vendor_dir}
 
+%set_cross_aws_lc_rs_env \
+  %write_cross_cmake_toolchain_conf \
+  export CMAKE_TOOLCHAIN_FILE="${PWD}/%{_cross_cmake_toolchain_conf}" \
+  export BINDGEN_EXTRA_CLANG_ARGS="--target=%{_cross_target} --sysroot=%{_cross_rootdir}" \
+  %{nil}
```

Comment:
I don't love this, since it's hard to imagine this combination being useful for anything other than `aws-lc-rs`, and these "shared" macros are meant to be reusable across specs.

Granted, you will need to build `aws-lc-rs` in multiple spec files, but you're still doing two unrelated things: configuring the `cmake` env and configuring the `bindgen` env.

It might be better to have two macros here:
```
%set_cross_cmake_env
%set_cross_bindgen_env
```

You could then have a third (in `macros/rust`) that calls them both, that could be used as a common preamble in packages that build Rust code, and we could add other global environment overrides there over time:
```
%set_cross_rust_env
```